onadata/libs/serializers/user_profile_serializer.py

```diff
@@ -253,14 +255,16 @@ def create(self, validated_data):
 
         created_by = request.user
         created_by = None if created_by.is_anonymous() else created_by
+        metadata['last_password_edit'] = timezone.now().isoformat()
```

Comment:
This is being updated everytime there is profile update, not everytime there was a password change.

Reply:
Actually it's every time an account is created not every time there's an account update.